docs/using-flatpak.rst

```diff
@@ -151,17 +179,17 @@ add it before the application is installed.
 Since Flatpak 1.2, the ``install`` command can search for applications. A
 simple::
 
- $ flatpak install gimp
+ $ flatpak install firefox
```

Comment:
I don't understand why these are changed.

Reply:
My reasoning is that GIMP uses outdated versions of some libraries (iirc), and more people would use Firefox on Linux than not. The word gimp also has negative connotations, which are unrelated to the software.